inc/git-updater/class-provider.php

```diff
@@ -93,7 +93,7 @@ public function get_package_metadata( DID $did ) {
 		$data->type = 'wp-' . $package->type;
 		$data->name = $package->name;
 		$data->slug = $package->slug;
-		$data->file = $package->file;
+		$data->filename = $package->file;
```

Comment:
Been roasting in the sun today so I haven't looked into this to answer it yet. Should `$package->file` change to `filename`?

Reply:
`$package` comes from GU, it is correct.